s1.html

```
@@ -0,0 +1,2 @@
+<div onclick="handleClick()">Submit</div>  <!-- 内联事件 → 改用 addEventListener -->
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)